partiql-spi/src/main/kotlin/org/partiql/spi/function/builtins/AggAvg.kt

```diff
@@ -10,22 +10,22 @@ import org.partiql.types.PType
 
 internal val Agg_AVG__INT8__INT8 = Aggregation.static(
     name = "avg",
-    returns = PType.decimal(),
+    returns = PType.decimal(38, 19),
```

Comment:
Ah, yes. I discussed with @jpschorr about what the appropriate precision/scale of average should be. It comes down to the fact that `AVG(x)` = `SUM(x) / COUNT(x)`. Since `COUNT` returns a BIGINT (JVM long) (which has a decimal precision of 19). We were originally failing conformance tests with AVG since the declared output type was a decimal with scale 0, so the average of 2 and 3 was returning 2 rather than 2.5. So, we opted for a max precision (38) followed by the precision of BIGINT for the division. It needs to be verified via RFC, but for now, we need to retain the fractional parts to some degree.

Reply:
I see. Perhaps we could include a TODO in the code and/or cut an issue?

Code-style wise, I think you should add some private top-level variable that can be reused by the AVG function instances that return `PType.decimal(38, 19)`.

Reply:
Added!